spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/writer/WriteConf.scala

```diff
@@ -122,6 +122,12 @@ object WriteConf {
       | per single core in MB/s. <br> Limit this on long (+8 hour) runs to 70% of your max throughput
       | as seen on a smaller job for stability""".stripMargin)
 
+  val TtlParam = ConfigParameter[Int] (
```

Comment:
My gut is also telling me this should be [Option[TTLOption]] with a default of None ... or possibly  we add another member to TTLOption which is None.